docs/known_issues.rst

```diff
@@ -139,6 +139,23 @@ A workaround for this at the moment would be to do::
 
 As well as with `~numpy.full` one cannot do `~numpy.zeros`, `~numpy.ones`, and `~numpy.empty`.
 
+The `~np.arange` function does not work either::
```

Comment:
Oops, should have noticed this while reviewing: this should be `~numpy.arange` (and once more below, plus `~numpy.linspace`)

Reply:
Should be fixed now.